htdocs/societe/list.php

```diff
@@ -672,6 +672,8 @@
 }
 $sql .= $hookmanager->resPrint;
 // Add GroupBy from hooks
+//removing PHP8 warnings
+$all, = isset($all,) ? $all, : '';
```

Comment:
Syntax error here. I suggest to fix by replacing the
$parameters = array('all' => $all, 'fieldstosearchall' => $fieldstosearchall);
into
$parameters = array('fieldstosearchall' => $fieldstosearchall);
just after.
The $all is never defined, so providing this parameter is useless.

Reply:
Ok